### ISSUE TITLE
Optimal data for the frequency method?

### ISSUE THREAD

**Issue by mniku** [benjjneb/decontam#33]
I’m slightly uncertain which DNA measures are optimal for the frequency based method (especially in case of samples containing animal tissues, where the proportion of microbial to animal DNA is often small and/or highly variable):

- 16S qPCR measurements of the original DNA samples

- DNA concentrations of the PCR products generated for Illumina sequencing (1st round / 2nd round PCR)

Obviously only the qPCR data tells the actual original amounts of starting material. But on the other hand, there are many steps between this and the final sequence data, so that the final DNA amounts used in the sequencing are something completely different. 

How should we evaluate the applicability of frequency based method in specific cases? Such as, how high read counts in negative controls vs. actual samples are acceptable for the statistics?

**Comment by benjjneb** [comment]
We believe that both types of DNA quantitation data will work. We have more testing using the DNA concentration post-PCR and prior to sequencing, simply because that data is more often available as it is generated "for free" as part of the usual sequencing workflows anyway. But in the more limited testing on qPCR data the method still seems to work, and [other publications report strong patterns of inverse frequency of contaminants using qPCR data](ttps://doi.org/10.1186/s12866-016-0689-4) - which is the pattern the frequency method relies on.

> How should we evaluate the applicability of frequency based method in specific cases? Such as, how high read counts in negative controls vs. actual samples are acceptable for the statistics?

The simplest and most useful evaluation is to inspect the distribution of scores assigned by the method. The expectation is that there will be a strong mode at low scores. In the cleanest cases the distribution will be clearly bimodal, while in other datasets the high-score mode is more wide and diffuse. However, the low-score mode should be there, and should be used to set the P* score threshold for identifying contaminants.

Another method is to simply inspect a few of the identified contaminants using the `plot_frequency` function.

**Comment by mniku** [comment]
Thanks, this is now clear!

**Comment by molboss** [comment]
Thank you for your assistance!

**Comment by rturba** [comment]
How would the pre-pooling DNA concentration be valuable if samples are pooled to be equimolar? Would using that data still make sense? I ran my samples together with other people that have different equimolar concentrations. In that case, would that be a more valuable data to use? Thank you!

**Comment by benjjneb** [comment]
@rturba

> How would the pre-pooling DNA concentration be valuable if samples are pooled to be equimolar? Would using that data still make sense?

Yes. Pre-pooling DNA concentrations still track the fraction of the sample reads that derive from teh sample vs. from contaminants.

> I ran my samples together with other people that have different equimolar concentrations. In that case, would that be a more valuable data to use?

Probably not. For identifying contaminants, you want to use samples that shared the same sample preparation history. Samples that were prepared differently will typically just be noise.
